src/storage-client/src/storage_collections.rs

```diff
@@ -137,166 +111,10 @@ pub trait StorageCollections: Debug {
         id: Vec<GlobalId>,
     ) -> Result<Vec<CollectionFrontiers<Self::Timestamp>>, StorageError<Self::Timestamp>>;
 
-    /// Atomically gets and returns the frontiers of all active collections.
-    ///
-    /// A collection is "active" when it has a non empty frontier of read
-    /// capabilties.
-    fn active_collection_frontiers(&self) -> Vec<CollectionFrontiers<Self::Timestamp>>;
-
     /// Checks whether a collection exists under the given `GlobalId`. Returns
     /// an error if the collection does not exist.
     fn check_exists(&self, id: GlobalId) -> Result<(), StorageError<Self::Timestamp>>;
 
-    /// Returns aggregate statistics about the contents of the local input named
```

Comment:
I also meant for these `snapshot_` methods to not be on the storage controller, long term, only on collections